CHANGELOG.md

```diff
@@ -2,23 +2,98 @@
+  The following metrics will be enabled by default "vcenter.host.network.packet.drop.rate",
+  "vcenter.vm.cpu.readiness", "vcenter.host.cpu.capacity", and "vcenter.host.cpu.reserved".
 
 ### 🚩Deprecations 🚩
 
-- (Splunk) Deprecate usage of bare environment variables and config sources in configuration files. 
+- (Splunk) Deprecate usage of bare environment variables and config sources in configuration files ([#5153](https://github.com/signalfx/splunk-otel-collector/pull/5153))
   - Use `${env:VAR}` or `${VAR}` instead of `$VAR`.
   - Use `${uri:selector}` instead of `$uri:selector`, e.g. `${file:/path/to/file}` instead of `$file:/path/to/file`.
 
 ### 💡 Enhancements 💡
-
-- (Splunk) `discovery`: Enable discovery mode for SQL Server receiver ([#5109](https://github.com/signalfx/splunk-otel-collector/pull/5109))
-- (Splunk) Update bundled OpenJDK to [11.0.24_8](https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.24%2B8) ([#5113](https://github.com/signalfx/splunk-otel-collector/pull/5113))
-- (Splunk) Rabbitmq receiver was added into the discovery feature([#5113](https://github.com/signalfx/splunk-otel-collector/pull/5051))
+- (Splunk) Auto Discovery for Linux:
+  - Bring SQL Server receiver into the discovery mode ([#5109](https://github.com/signalfx/splunk-otel-collector/pull/5109))
+  - Bring Cassanda JMX receiver into the discovery mode ([#5112](https://github.com/signalfx/splunk-otel-collector/pull/5112))
+  - Bring Rabbitmq receiver into the discovery mode ([#5051](https://github.com/signalfx/splunk-otel-collector/pull/5051))
+  - Reuse discovery receiver's obsreport for receivercreator ([#5111](https://github.com/signalfx/splunk-otel-collector/pull/5111))
+- (Splunk) Update bundled OpenJDK to [11.0.24_8](https://github.com/adoptium/temurin11-binaries/releases/tag/jdk-11.0.24%2B8) ([#5113](https://github.com/signalfx/splunk-otel-collector/pull/5113), [#5119](https://github.com/signalfx/splunk-otel-collector/pull/5119))
+- (Splunk) Upgrade github.com/hashicorp/vault to v1.17.2 ([#5089](https://github.com/signalfx/splunk-otel-collector/pull/5089))
+- (Splunk) Upgrade github.com/go-zookeeper/zk to 1.0.4 ([#5146](https://github.com/signalfx/splunk-otel-collector/pull/5146))
+- (Core) `configtls`: Mark module as stable. ([#9377](https://github.com/open-telemetry/opentelemetry-collector/pull/9377))
+- (Core) `confmap`: Remove extra closing parenthesis in sub-config error ([#10480](https://github.com/open-telemetry/opentelemetry-collector/pull/10480))
+- (Core) `configgrpc`: Update the default load balancer strategy to round_robin ([#10319](https://github.com/open-telemetry/opentelemetry-collector/pull/10319))
+  To restore the behavior that was previously the default, set `balancer_name` to `pick_first`.
+- (Core) `cmd/builder`: Add go module info the builder generated code. ([#10570](https://github.com/open-telemetry/opentelemetry-collector/pull/10570))
```

Comment:
We don't have any dependency on the build, so we can safely remove this.

Reply:
removed